src/main/java/com/sse/ooseproject/models/Employee.java

```diff
@@ -11,6 +11,8 @@ public class Employee extends Person {
     @ManyToOne
     @JoinColumn(name = "university_id")
     private University university;
+    @OneToOne(mappedBy = "employee")
```

Comment:
glaube das müsste nicht `(mappedBy = "employee"` sondern `(mappedBy = "chairOwner")` sein. Also zumindest wenn ich das bei [Baeldung ](https://www.baeldung.com/jpa-joincolumn-vs-mappedby#mappedBy) richtig lese. Wahrscheinlich sollten wir wenn wir die aufgaben für den sprint fertig haben einmal alle mappedBy sachen überprüfen ob die richtig sind. kann sein dass ich das auch nicht richtig gemacht habe.

Reply:
Aber es gibt keine Tabelle "chairOwner". Dachte bei mappedBy wird quasi die Tabelle angegeben, in der die ID zufinden ist, welche in der Spalte der JoinColumn Annotation in chair angegeben ist.

Reply:
Hmm, aber ja, dass mit dem Attribut macht eigentlich mehr Sinn. Dann ändere ich das mal.

Reply:
laut der Baeldung Seite ist das was bei mappedBy drin steht der name des Attributs dass die relation auf der anderen seite angibt. Deswegen hätte ich jetzt gesagt chairOwner (wusste das vor gestern auch nicht)

> Here, the value of mappedBy is the name of the association-mapping attribute on the owning side. With this, we have now established a bidirectional association between our Employee and Email entities.